backend/tests/functional_tests/scripts/test_sync_configuration.py

```diff
@@ -51,6 +53,8 @@ def test_sync_with_empty_database(prepare_environment: Any) -> None:
     kidneys_segmentation = TasksRepository.get_task_by_key('KIDNEYS_SEGMENTATION')
     assert kidneys_segmentation.name == 'Kidneys segmentation'
     assert kidneys_segmentation.image_path == 'assets/icon/kidneys_dataset_icon.svg'
+    assert kidneys_segmentation.description == 'This is a test task'
+    assert kidneys_segmentation.label_examples == ['assets/example_1', 'assets/example_2']
```

Comment:
This is wrong way to compare list. PostgreSQL may return this list in different order.

Safer way to do this:
```python
assert set(kidneys_segmentation.label_examples) == {'assets/example_1', 'assets/example_2'}
```